plugins/modules/system/listen_ports_facts.py

```diff
@@ -72,11 +84,21 @@
       returned: if TCP servers were found
       type: list
       contains:
-        address:
+        local_address:
```

Comment:
The old name needs to stay, otherwise this is a breaking change.

Reply:
okay, then will stick to the old name.